### ISSUE TITLE
Simple Example

### ISSUE THREAD

**Issue by v4hn** [bit-bots/particle_filter#2]
It would be great to have at least one example node that generates a ground truth motion/observation (e.g. using sine waves) and sets up the filter to approximate this.

Without such an example the reader basically has to look through all headers to figure out where to start...

**Comment by NFiedler** [comment]
You are right. Currently i don't have the time to create such an example, but the [libPF](https://github.com/stwirth/libPF) library (on which this project is based) offers some examples of the filter in use. They might help understanding it for now.
